cli/controller.js

```diff
@@ -0,0 +1,82 @@
+#! /usr/bin/env node
+
+//Import required modules
+const hosts = require("../models/hosts")
+const views = require("./views")
+
+//set command variable
+const command = process.argv[2];
+
+(async function(command){
```

Comment:
Ths async IEFE belongs in `bin/cli` and the controller should export all the functionality.